src/constants/pools.ts

```diff
@@ -128,3 +128,11 @@ const doubleEstimatePoolsKovan =
 export const DOUBLE_ESTIMATE_POOLS = doubleEstimatePools
 export const DOUBLE_ESTIMATE_POOLS_MATIC = doubleEstimatePoolsMatic
 export const DOUBLE_ESTIMATE_POOLS_KOVAN = doubleEstimatePoolsKovan
+
+// NOTE: add the pool address and its first deposit tx here for other chain
+export const DEPOSIT_TXHASH: Record<string, string> = {
```

Comment:
@geonellov so this means base apr on the UI will only show up on mainnet right?

I guess it's fine for now @ctverceles ?

Reply:
@geonellov @schystz we want it to show up on all networks

Reply:
ok @geonellov can we pls add this in the README? @tracyarciaga @schystz we'll need to take note of this so we add it for every network when we merge this

Reply:
> ok @geonellov can we pls add this in the README? @tracyarciaga @schystz we'll need to take note of this so we add it for every network when we merge this

Updated README